### ISSUE TITLE
problem: "TgtgCaptchaError"

### ISSUE THREAD

**Issue by danielblogo89** [Der-Henning/tgtg#342]
Hi Henning,

since today i've got the following error code:
(<class 'models.errors.TgtgCaptchaError'>, TgtgCaptchaError(403, b'<html><head><title>apptoogoodtogo.com</title><style>#cmsg{animation: A 1.5s;}@keyframes A{0%{opacity:0;}99%{opacity:0;}100%{opacity:1;}}</style></head><body style="margin:0"><p id="cmsg">Please enable JS and disable any ad blocker</p>

Do you know how to fix it?

all the best

**Comment by danielblogo89** [comment]
now i have tested it with the newest scanner version and got this error: 

2023-05-25 17:06:28 WARNING  Too many captcha Errors! Sleeping for 10 minutes...

**Comment by djan271991** [comment]
"I have the same problem today

**Comment by ArcCdr** [comment]
FYI, same issue for me as well.

**Comment by nikoonline3** [comment]
I got a capcha request with an image on my smartphone when I launched the app.

"A bot has been found on your network..."

**Comment by antonioli86** [comment]
Same here. :(

**Comment by danielblogo89** [comment]
> Same here. :( Possible solution: [ahivert/tgtg-python#241 (comment)](https://github.com/ahivert/tgtg-python/issues/241#issuecomment-1544533679) ?

too technical for me :((((

**Comment by Der-Henning** [comment]
Hi.
I can confirm the observed behavior. I don't have a solution at the moment, but I am looking into it.

@antonioli86 The datadome cookie is already implemented.

I will give you an update if I find a solution.
Feel free to post any possible solutions or workarounds.

**Comment by danielblogo89** [comment]
I have testet is even with my phones wifi but it did not work either...

**Comment by Der-Henning** [comment]
Update:
TGTG changed their API. A small update 12dbc576c40b593a1229a73515fbcd62cf037782 should do the trick. I will create a new release as soon as possible.

**Comment by pershoot** [comment]
@Der-Henning Built from dev; received initial test and another notif. right after that.  I'll monitor.  Thank you.

**Comment by Der-Henning** [comment]
I created a new release v1.16.3 containing the new API endpoint as a hotfix as it seems to work for now. Please note if you have any problems or notice any strange behavior. Thank you.

**Comment by antonioli86** [comment]
> I created a new release v1.16.3 containing the new API endpoint as a hotfix as it seems to work for now. Please note if you have any problems or notice any strange behavior. Thank you.

Will push new image in Docker Hub?

**Comment by Der-Henning** [comment]
@antonioli86 Yes the CI/CD takes some minutes to create the release binaries and docker images. But it just completed.

**Comment by antonioli86** [comment]
> @antonioli86 Yes the CI/CD takes some minutes to create the release binaries and docker images. But it just completed.

Thanks. Just did pull and seems all fine now. Thanks!

**Comment by danielblogo89** [comment]
> I created a new release v1.16.3 containing the new API endpoint as a hotfix as it seems to work for now. Please note if you have any problems or notice any strange behavior. Thank you.

Thank you, but on my end it seems to not work 
<img width="571" alt="Bildschirmfoto 2023-05-25 um 21 45 34" src="https://github.com/Der-Henning/tgtg/assets/134615336/10a91d83-316f-4fec-9db9-839043f5b187">

**Comment by Der-Henning** [comment]
@danielblogo89 
Sometimes it helps to relogin to the TGTG API. To do so delete the saved tokens, userid and datadome cookie. Or you have to wait for some time.

**Comment by nikoonline3** [comment]
for docker I need to use a specific tag "_image: derhenning/tgtg:v1.16.3_". With latest I have version 1.16.2.

**Comment by Der-Henning** [comment]
Hi @nikoonline3.
You have to make sure that you repulled the latest image. Otherwise docker uses the current local image with the specified tag. 
Using docker compose you can update the local image with `docker-compose pull` or use `docker pull derhenning/tgtg:latest`.

**Comment by Der-Henning** [comment]
@danielblogo89 Is it working for you now?

**Comment by karimhajji12** [comment]
I have the same problem that @danielblogo89. Maybe my IP is blocked, because there's this error (too many captchas) only with my personal Wifi, when is switch my internet connection there's no error

**Comment by SergiBaucells** [comment]
The same thing happens to me as to @karimhajji12 . The same computer on the network that has always been configured and working gives the captcha error and if I change the network it doesn't work for me.

![Captura de pantalla 2023-06-02 090714](https://github.com/Der-Henning/tgtg/assets/23049517/a3dfc0d5-70f2-4de3-9a1b-18f581dd0e18)

**Comment by SergiBaucells** [comment]
Extra information: without touching anything, it has been working for 1 day, but now I get the 403 problem again.

**Comment by Zilverdael80** [comment]
I have the same issue, running latest version [1.16.3](https://github.com/Der-Henning/tgtg/releases/tag/v1.16.3)
![image](https://github.com/Der-Henning/tgtg/assets/116879022/9b896b70-2b4b-47ac-a691-a6786fb03254)
